lib/exercises.rb

```diff
@@ -1,19 +1,62 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n * s * log(s)) , where s = string length
+# Space Complexity: O(n)
+
+
 
 def grouped_anagrams(strings)
```

Comment:
👍 Well done, if you assume the strings are short, you can even consider this one O(n).